### DISCUSSION TITLE
Chrome extension is not working

### DISCUSSION THREAD

**Discussion by tharushika-jeewantha** [subhra74/xdm#964] [Q&A]
Hi guys,
XDM Chrome extension is not working. 
XDM Version = 7.2.11

In the firefox browser, it works fine. but I want to use it in chrome

**Comment by frank392a** [comment]
It works perfect on Edge and I think Firefox

**Comment by subhra74** [comment]
Please wait for next stable version or you can try the beta version which is currently being developed:
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta

**Comment by tharushika-jeewantha** [comment]
Thanks
